hubblestack/extmods/module_runner/audit_runner.py

```diff
@@ -84,7 +84,7 @@ def _execute(self, audit_data_dict, audit_file, args):
                 })
                 log.error(herror)
             except Exception as exc:
-                log.exception(exc)
+                log.error(exc)
```

Comment:
Make sure it is printing the stacktrace

Reply:
yes it will print the stacktrace in this form as well.